sdks/python/apache_beam/ml/inference/base.py

```diff
@@ -836,7 +838,12 @@ def get_metrics_collector(self, prefix: str = ''):
 
   def setup(self):
     self._metrics_collector = self.get_metrics_collector()
-    self._model_handler.set_environment_vars()
+    try:
+      self._model_handler.set_environment_vars()
+    except AttributeError:
+      logging.warning(
+          "ModelHander should call super().__init__ or set _env_vars.")
```

Comment:
We are checking `if '_env_vars` in self.__dict__ in ModelHandler's set_environment_vars, if that attribute is not in the dict, then we skip it right? 
But we expect that case to show warning right but it actually doesn't

Reply:
Yes, I removed this catch.